.github/workflows/labeler.yml

```diff
@@ -0,0 +1,11 @@
+name: "Pull Request Labeler"
+on:
+- pull_request_target
```

Comment:
This one is not working now. (Ref: https://github.com/cncf/glossary/pull/471#issuecomment-1049506581)

I suggest:

```yaml
on:
  pull_request:
    types: [opened]
```

Ref: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request

Reply:
made the suggested changes